cooltools/api/expected.py

```diff
@@ -899,6 +900,141 @@ def blocksum_pairwise(
+    suffix : string, optional
+        If provided, use the specified string as the suffix of the output column's name
+
+    Returns
+    -------
+    cvd : pandas.DataFrame
+        A cvd table with extra column for the log-smoothed contact frequencies (by default, "count.avg.smoothed.agg" or "balanced.avg.smoothed.agg" depends on whether balanced or not).
+
+    Notes
+    -----
+    Parameters in "cols" will be used:
+
+    dist:
+        Indicate the name of the column that stores distance values (by default, "dist").
+    n_pixels:
+        Indicate the name of the column that stores number of pixels (by default can be either "n_total" or "n_valid" depends on whether balanced or not).
+    n_contacts:
+        Indicate the name of the column that stores the sum of contacts (by default can be either "count.sum" or "balanced.sum" depends on whether balanced or not).
+    output_prefix:
+        Indicate the name prefix of the column that will store output value (by default, can be either "count.avg" or "balanced.avg" depends on whether balanced or not).
+    """
+    cvd_smoothed = expected_smoothing._smooth_cvd_group(
+        cvd,
+        sigma_log10=sigma_log10,
+        window_sigma=window_sigma,
+        points_per_sigma=points_per_sigma,
+        cols=cols,
+        suffix=suffix,
+    )
+
+    # add aggeragated smoothed columns to the result
+    cvd = cvd.merge(
+        cvd_smoothed[[cols["output_prefix"] + suffix, cols["dist"]]],
+        on=[cols["dist"]],
+        how="left",
+    )
+
+    return cvd
+
+
+def per_region_smooth_cvd(
+    cvd,
+    sigma_log10=0.1,
+    window_sigma=5,
+    points_per_sigma=10,
+    cols=None,
+    suffix="",
+):
+    """
+    Smooth the contact-vs-distance curve for each region in log-space.
+
+    Parameters
+    ----------
+    cvd : pandas.DataFrame
+        A dataframe with the expected values in the cooltools.expected format.
+    sigma_log10 : float, optional
+        The standard deviation of the smoothing Gaussian kernel, applied over log10(diagonal), by default 0.1
+    window_sigma : int, optional
+        Width of the smoothing window, expressed in sigmas, by default 5
+    points_per_sigma : int, optional
+        If provided, smoothing is done only for `points_per_sigma` points per sigma and the
+        rest of the values are interpolated (this results in a major speed-up). By default 10
+    cols : dict, optional
+        If provided, use the specified column names instead of the standard ones.
+        See DEFAULT_CVD_COLS variable for the format of this argument.
+    suffix : string, optional
+        If provided, use the specified string as the suffix of the output column's name
+
+    Returns
+    -------
+    cvd : pandas.DataFrame
+        A cvd table with extra column for the log-smoothed contact frequencies (by default, "count.avg.smoothed" or "balanced.avg.smoothed" depends on whether balanced or not).
```

Comment:
see suggestion above to mention 'balanced.avg.smoothed' first then 'count.avg.smoothed' as most users would use balanced data